app/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(15, 3)) == 15
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    parts = split_integer(12, 3)
+    assert len(parts) == 3
+    assert max(parts) - min(parts) <= 1
+    assert parts == [4, 4, 4]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    parts = split_integer(8, 1)
+    assert len(parts) == 1
+    assert parts[0] == 8
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    parts = split_integer(17, 4)
+    assert len(parts) == 4
+    assert max(parts) - min(parts) <= 1
+    assert parts == [4, 4, 4, 5]
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    parts = split_integer(4, 6)
+    assert len(parts) == 6
+    assert max(parts) - min(parts) <= 1
+    assert parts == [0, 0, 1, 1, 1, 1]
```

Comment:
do not complicate your tests, read the checklist and do them with only one expression in every test